action.rb

```diff
@@ -1,25 +1,35 @@
 class Action
   attr_reader :headers, :body, :request
 
-  def initialize &block
+  def initialize(&block)
     @block = block
-    @status = 200
-    @headers = {'Content-Type' => 'text/html'}
-    @body = ''
+    @headers = {'Content-Type' => 'application/json'}
   end
 
-  def status value = nil
-    value ? @status = value : @status
+  def status(value = nil)
+    value ? @status = value : @status || 200
```

Comment:
@brandondees also no raw response at all. Seems "empty" however response is successful with the same amount of bytes no matter the `Content-Type`. A mystery indeed!

http://devpunks.herokuapp.com

![capture d ecran 2017-01-26 a 00 27 43](https://cloud.githubusercontent.com/assets/38223/22320901/4810a6cc-e35e-11e6-922d-a3c0b830420e.png)
![capture d ecran 2017-01-26 a 00 27 35](https://cloud.githubusercontent.com/assets/38223/22320902/481374b0-e35e-11e6-9806-844de00bb1b8.png)